services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java

```diff
@@ -45,7 +45,7 @@ public InternalDPMResultList createNetworkConfiguration(@RequestBody NetworkConf
         return dpmService.createNetworkConfiguration(networkConfiguration);
     }
 
-    @PostMapping({"/network-configuration", "v4/network-configuration"})
+    @PutMapping({"/network-configuration", "v4/network-configuration"})
```

Comment:
Do we revert the change in https://github.com/futurewei-cloud/alcor/commit/971f0b602c5dbe645a28ca5f8659ffc7d7b24e7a?

Reply:
No, I think I reuse my branch for this PR.